### PR TITLE
Update BCD info, part 9

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Adding to #19185 

The PR updates BCD info in WebAPI docs.
It focuses on files that have only left behind 'experimental' tags and headers.

### CODE REVIEW

File: files/en-us/web/api/inputdevicecapabilities/index.md [hunk 1]
```diff
@@ -4,13 +4,12 @@ slug: Web/API/InputDeviceCapabilities
...
 tags:
   - API
-  - Experimental
   - InputDeviceCapabilities
   - Interface
   - Reference
 browser-compat: api.InputDeviceCapabilities
 ---
-{{APIRef("InputDeviceCapabilities API")}}{{SeeCompatTable}}
+{{APIRef("InputDeviceCapabilities API")}}
```
Comment (teoli2003) [inline files/en-us/web/api/inputdevicecapabilities/index.md:12]: Given only Chromium supports it, it should be marked experimental on MDN (and in BCD).

File: files/en-us/web/api/inputdevicecapabilities_api/index.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ tags:
   - Reference
 browser-compat: api.InputDeviceCapabilities
 ---
-{{DefaultAPISidebar("InputDeviceCapabilities API")}}{{SeeCompatTable}}
```
Comment (teoli2003) [inline files/en-us/web/api/inputdevicecapabilities_api/index.md:12]: Ditto

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
👍

**Comment by OnkarRuikar** [comment]
Should we delete deprecated features that have 0 bowser support from mdn/content?
For example, `ptime` and `codecPayloadType` on page [RTCRtpEncodingParameters](https://developer.mozilla.org/en-US/docs/Web/API/RTCRtpEncodingParameters).

They've been [marked deprecated](https://github.com/mdn/browser-compat-data/pull/17301), waiting for next release.

**Comment by teoli2003** [comment]
If a feature is not implemented in any browser (and not on the verge of _coming to life_), it should be removed from both bcd and MDN.

In the wiki era of MDN, we have no way of keeping these features in a Draft PR, so we had to publish them. Today we wouldn't publish them.

**Pull request merged**